sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_utils.py

```diff
@@ -12,6 +12,18 @@
 _ID_PREFIX = "Id="
 _SECRET_PREFIX = "Secret="
 
+# Azure Configuration audience URLs
+_AZURE_PUBLIC_CLOUD_AUDIENCE = "https://appconfig.azure.com"
+_AZURE_US_GOVERNMENT_AUDIENCE = "https://appconfig.azure.us"
+_AZURE_CHINA_AUDIENCE = "https://appconfig.azure.cn"
+_DEFAULT_SCOPE_SUFFIX = "/.default"
```

Comment:
What do you think about storing the scopes directly, instead of storing the audience and then appending the scope every time?

Reply:
I made a slight update what do you think. I keep it separate, but I append it to it in the constant.